### ISSUE TITLE
URL in address bar is incorrect

### ISSUE THREAD

**Issue by arfaRed** [browserstate/ajaxify#26]
Hi,

I am using this plugin for one my client's site at http://test.kittysu.com
Things work as intended on major browsers.
But on window's IE9 browser, I get the following issues
1. # gets added between site's root URL and the new page
2. When you click on the homepage link(click on logo) it adds an suid url parameter
I checked one of the sites mentioned in showcase for history.js(http://craftedpixelz.co.uk/), and that is not appending any # tags

Can you please guide me how to sort the above mentioned issues.

Thanks

**Comment by Jonathlee** [comment]
Agree, hopefully someone can provide any method to disable ajaxify script if browser is ie9 and below.
